cms/templates/container.html

```diff
@@ -100,9 +100,14 @@
                             <div class="nav-sub">
                                 <ul>
                                 % for child in block['children']:
-                                <li class="nav-item">
-                                    <a href="${xblock_studio_url(child)}">${child.display_name_with_default}</a>
-                                </li>
+                                    <%
+                                        url = xblock_studio_url(child)
+                                    %>
+                                    % if url:
```

Comment:
In what case is there no url?

Reply:
When in nested xblock page, links to other sibling xblocks (ex: HTML Block) will be ``None`` due to the code [here](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/helpers.py#L104).

Reply:
I've also added a comment specifiying the need of the check there.